cmd/agent/common/helpers.go

```diff
@@ -49,3 +52,46 @@ func setupConfig(confFilePath string, configName string, withoutSecrets bool) er
 	}
 	return nil
 }
+
+// SetupSystemProbeConfig reads the system-probe.yaml into the global config object
+func SetupSystemProbeConfig(sysProbeConfFilePath string) error {
+	var (
+		file *os.File
+		err  error
+	)
+
+	// Open the system-probe.yaml file if it's in a custom location
+	if len(sysProbeConfFilePath) != 0 {
+		// If file is set directly,  assume we should use it
+		if strings.HasSuffix(sysProbeConfFilePath, ".yaml") {
+			// Open the file if they pass it in directly
+			file, err = os.Open(sysProbeConfFilePath)
+		} else {
+			file, err = os.Open(sysProbeConfFilePath + "/system-probe.yaml")
```

Comment:
This is not windows compatible, we should use [path.Join](https://golang.org/pkg/path/#Join)